packages/charts/src/components/LineChart/LineChart.spec.tsx

```diff
@@ -37,7 +38,9 @@ describe('LineChart', () => {
 
         const svgContainer = container.getElementsByTagName('svg')[0];
 
-        expect(svgContainer?.getAttribute('width')).toBe('400');
+        console.log(container.attributes);
```

Comment:
still debugging?

Reply:
you are just too fast ;p

Reply:
![image](https://github.com/user-attachments/assets/ce6104b2-6d73-4ae4-86b1-a47e102f2789)